app/workers/backup_worker.rb

```diff
@@ -9,8 +9,12 @@ class BackupWorker
     backup_id = msg['args'].first
 
     ActiveRecord::Base.connection_pool.with_connection do
-      backup = Backup.find(backup_id)
-      backup&.destroy
+      begin
+        backup = Backup.find(backup_id)
+        backup.destroy
+      rescue ActiveRecord::RecordNotFound
+        true
+      end
```

Comment:
Is this related to this PR? :thinking:

Reply:
No, but there was no point doing `backup&.destroy` after `find` because `find` raises an error.